newIDE/app/src/Profile/CurrentUsageDisplayer.js

```diff
@@ -101,21 +161,34 @@ const CurrentUsageDisplayer = ({
               !isFeatureLocked ? 'Build limit reached' : 'Unlock build type'
             }
             label={<Trans>Upgrade your subscription</Trans>}
-            makeButtonRaised={remainingBuilds === 0}
+            makeButtonRaised
+            payWithCreditsOptions={
+              hidePurchaseWithCredits
+                ? undefined
+                : {
+                    label: (
+                      <Trans>Purchase with {usageCreditPrice} credits</Trans>
+                    ),
+                    onPayWithCredits: () => {
+                      onPurchaseBuildWithCredits();
+                    },
+                  }
+            }
+            hideButton={cannotUpgradeSubscription}
           >
             <Line>
               {!isFeatureLocked ? (
                 <Column noMargin>
+                  <Text noMargin>{remainingMultipleMessage}</Text>
                   <Text noMargin>
-                    {remainingBuilds === 1
-                      ? remainingSingleMessage
-                      : remainingMultipleMessage}
-                  </Text>
-                  <Text noMargin>
-                    <Trans>
-                      Need more power? Upgrade your GDevelop subscription to
-                      increase the limits.
-                    </Trans>
+                    {cannotUpgradeSubscription ? (
+                      <Trans>Use GDevelop credits to start an export.</Trans>
+                    ) : (
+                      <Trans>
+                        Use GDevelop credits or upgrade your subscription to
+                        increase the limits.
```

Comment:
to increase your quotas?

Reply:
I feel that quota is a bit harder to understand and also to translate, where limits is quite common